src/icons/Valora.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react'
+import { ViewStyle } from 'react-native'
+import Svg, { Path } from 'react-native-svg'
+
+interface Props {
+  width?: number
+  height?: number
+  style?: ViewStyle
+}
+
+export default function Valora({ width = 243, height = 64, style }: Props) {
+  return (
+    <Svg width={width} height={height} viewBox={`0 0 ${width} ${height}`} fill="none" style={style}>
```

Comment:
this needs a ViewBox if you want to update width and height with props

Reply:
Is that something different than the `viewBox={`0 0 ${width} ${height}`}` that I have? In other examples with this like Blob.tsx it is done the same as here.